apps/react-editor/index.d.ts

```diff
@@ -2,8 +2,8 @@ import { HTMLAttributes, Component } from 'react';
 import ToastuiEditor, { EditorOptions } from '@toast-ui/editor';
 import ToastuiEditorViewer, { ViewerOptions } from '@toast-ui/editor/dist/toastui-editor-viewer';
 
-type EditorProps = EditorOptions & HTMLAttributes<HTMLElement>;
-type ViewerProps = ViewerOptions & HTMLAttributes<HTMLElement>;
+type EditorProps = Omit<EditorOptions, 'el'> & HTMLAttributes<HTMLElement>;
+type ViewerProps = Omit<ViewerOptions, 'el'> & HTMLAttributes<HTMLElement>;
```

Comment:
혹시.. `HTMLAttributes<HTMLElement>`가 사용된 이유 알 수 있을까요??

Reply:
@seonim-ryu 
리액트 랩퍼에 공통으로 적용된 타입 정의로 알고 있습니다. 리액트 JSX 엘리먼트에 props 로 받은 `attribute`들을 설정하기 위해 넘겨주는 속성 같은데..에디터 같은 경우는 저 속성을 JSX 엘리먼트에 아예 넘겨주질 않아서 사실상 의미가 없을 것 같습니다.

Reply:
넵. 그럼 에디터는 제거하도록 할게요.

Reply:
Why was 'el' removed in the following commit #[feat: remove type of html attribute](https://github.com/nhn/tui.editor/pull/856/commits/3afe6ef010efe869be02fffa98a76a8664d30fd9)
I was using onChange to detect changes to the text and automatically populate an object.
```
<Editor 
  ...
  onChange={}
/>
```
to get the current value, but because it was removed you have to get the value programtically before you save the content or something.

Another way of having all props available, but 'el' optional is
`type EditorProps = Pick<Partial<EditorOptions>, 'el'> & Omit<EditorOptions, 'el'> `
I can make a PR if needed, but I was curious to why was it removed in the first place. Is there an alternative?